driver-adapters-wasm/neon-http-vercel-nextjs-edgemw/middleware.js

```diff
@@ -25,6 +25,13 @@ async function getResponse() {
         name: true,
       },
     }),
+    create2: await prisma.user.create({
```

Comment:
```suggestion
      // Since `createMany` does not work we add a second create 
      // the record is used for `findUniqueOrThrow(...)` below
    create2: await prisma.user.create({
```